tests/yakus/dragonpon.test.cpp

```diff
@@ -0,0 +1,105 @@
+#include <gtest/gtest.h>
```

Comment:
Might be best to test the wind and dragon pon in the same file since they are the same function.

Reply:
I felt it made more sense to keep then separate as there is a larger number of tests per each situation. Regardless of how the scoring is implemented behind the scenes, there is a file per yaku type (so there's dragon pon and wind pons as separate categories).

Reply:
I believe they should be kept separate due to the quantity of tests. We should discuss this in gnv

Reply:
Sounds good just wanted to hear your reasoning.